ops/restimpl/src/main/java/org/sdo/iotplatformsdk/ops/config/OpsConfiguration.java

```diff
@@ -0,0 +1,333 @@
+import org.sdo.iotplatformsdk.common.protocol.config.SecureRandomFactory;
+import org.sdo.iotplatformsdk.common.protocol.security.AsymKexCodec;
+import org.sdo.iotplatformsdk.common.protocol.security.SignatureServiceFactory;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsAsymKexCodec;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsOwnerEventHandlerFactory;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsPropertiesLoader;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsProxyStorage;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsServiceInfoModule;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsSessionStorageFactory;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsSetupDeviceServiceInfo;
+import org.sdo.iotplatformsdk.ops.opsimpl.OpsSignatureServiceFactory;
+import org.sdo.iotplatformsdk.ops.rest.OpsControllerAdvice;
+import org.sdo.iotplatformsdk.ops.rest.OpsCustomSslContextFactory;
+import org.sdo.iotplatformsdk.ops.rest.OpsHealthController;
+import org.sdo.iotplatformsdk.ops.rest.OpsTransferOwnershipController;
+import org.sdo.iotplatformsdk.ops.rest.RestClient;
+import org.sdo.iotplatformsdk.ops.serviceinfo.ServiceInfoModule;
+import org.sdo.iotplatformsdk.ops.to2library.KeyExchangeDecoder;
+import org.sdo.iotplatformsdk.ops.to2library.Message255Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message40Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message42Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message44Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message46Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message48Handler;
+import org.sdo.iotplatformsdk.ops.to2library.Message50Handler;
+import org.sdo.iotplatformsdk.ops.to2library.OwnerEventHandler;
+import org.sdo.iotplatformsdk.ops.to2library.SessionStorage;
+import org.sdo.iotplatformsdk.ops.to2library.SetupDeviceService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ConcurrentTaskExecutor;
+import org.springframework.web.servlet.config.annotation.AsyncSupportConfigurer;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * Configuration class for the application that registers beans.
+ */
+@Configuration
+@EnableAutoConfiguration
+@EnableAsync
+public class OpsConfiguration implements WebMvcConfigurer {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(OpsConfiguration.class);
+
+  private ConcurrentTaskExecutor executor;
+
+  private final int threadPoolSize = null != OpsPropertiesLoader.getProperty("thread.pool.size")
+      ? Integer.parseInt(OpsPropertiesLoader.getProperty("thread.pool.size"))
+      : 10;
```

Comment:
Define the default values as a constant (used across different places).

Reply:
This is a one-time usage of this property and its value at the OPS.